starfish/image/_registration/fourier_shift.py

```diff
@@ -54,7 +86,9 @@ def run(self, image: ImageStack) -> ImageStack:
                     result = shift_im(data, shift)
                     image.set_slice(indices=indices, data=result)
 
-        return image
+        if not in_place:
+            return image
+        return None
```

Comment:
technically not required.

Reply:
Mypy got upset without it there at one point. I will check if it gets upset here, and if not will remove it.